scipy/signal/windows.py

```diff
@@ -1390,6 +1390,83 @@ def cosine(M, sym=True):
     return w
 
 
+def exponential(M, N=None, tau=1., sym=True):
+    r"""Return an exponential (or Poisson) window.
+
+    Parameters
+    ----------
+    M : int
+        Number of points in the output window. If zero or less, an empty
+        array is returned.
+    N : float, optional
+        Parameter defining the center location of the window function.
+        IF not given, then:
+          - for ``sym = True``: ``N = (M-1) / 2``.
+          - for ``sym = False``:  by default ``N = 0``.
+    tau : float, optional
+        Parameter defining the decay.  For ``N=0`` use ``tau = -(M-1) / ln(x)``
+        if ``x`` is the fraction of the window at the end.
+    sym : bool, optional
+        When True (default), generates a symmetric window, for use in filter
+        design.
+        When False, generates a periodic window, for use in spectral analysis.
+
+    Returns
+    -------
+    w : ndarray
+        The window, with the maximum value normalized to 1 (though the value 1
+        does not appear if `M` is even and `sym` is True).
+
+    Notes
+    -----
+    The Exponential window is defined as
+
+    .. math::  w(n) = e^{-|n-N| \frac{1}{\tau}}
```

Comment:
Probably should use \exp here.  @ev-br just had a PR to fix up various latex math expression like this. What do you think about not use \frac here? I don't think it adds enough value given the line noise it adds when reading the docs from a console.  (I suppose these two comments are pulling this expression in both directions at once...)